config/rbac/role/role.yaml

```diff
@@ -657,6 +657,23 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
+  - policy
+  resources:
+  - poddisruptionbudgets
+  verbs:
+  - create
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - policy
+  resources:
+  - poddistruptionbudgets
```

Comment:
Type caused by https://github.com/Kong/gateway-operator/pull/464/files#diff-59fdb2373e0a4c4eb9f34c451f482c01b9ef65679ccfd7290152a4488da21869R169.

This needs to be changed at the source and regenerated.